DemoApp/.env

```diff
@@ -0,0 +1,2 @@
+APP_DB=mongodb://localhost:27017/node-auth
+APP_SECRET=qwertyuiop
```

Comment:
Never upload the .env file on GitHub. Use 'dotenv' npm to access .env variables.